packages/main/src/Popover.ts

```diff
@@ -717,14 +743,35 @@ class Popover extends Popup {
 		return actualPlacement;
 	}
 
+	isInsideIframe() {
```

Comment:
Scrollbar is on the right side on Firefox.
Check why does it behave differently on Chromium based browsers only in iframes. Is this a known bug of Chromium?

Reply:
Unfortunately the Firefox behavior is the wrong one.
Here is open issue on the topic: https://bugzilla.mozilla.org/show_bug.cgi?id=619963
And also older report for Chromium: https://issues.chromium.org/issues/40322053